cicd/cmd/run-it-smoke-tests/main.go

```diff
@@ -59,6 +59,10 @@ func main() {
 		mvnFlags.IntegrationTestParallelism(4),
 		mvnFlags.StaticBigtableInstance("teleport"),
 		mvnFlags.StaticSpannerInstance("teleport"),
+		mvnFlags.StaticOracleInstance("10.128.0.90"),
+		mvnFlags.CloudProxyHost("10.128.0.34"),
+		mvnFlags.CloudProxyPort("33134"),
+		mvnFlags.CloudProxyPassword("t>5xl%J(&qTK6?FaZ"),
```

Comment:
I understand that this is internal network only so this "password" is fine. However can we put most flags into a config file, e.g. yaml file and read them on runtime? Currently it is hardcoded in source code in several places

Reply:
I refactored to use the flags module instead and set default values for the flags - config file at that point seemed like overkill but let me know